Bluepill-runner/BluepillRunnerTests/BPRunnerTests.m

```diff
@@ -30,6 +30,7 @@ - (void)setUp {
     NSString *hostApplicationPath = [BPTestHelper sampleAppPath];
     NSString *testBundlePath = [BPTestHelper sampleAppNegativeTestsBundlePath];
     self.config = [BPConfiguration new];
+    self.config.program = BP_MASTER;
```

Comment:
We need this for mutable copy of `BPConfiguration`, it should be fine since this is test code.